upgrade/upgrade-scripts/common.sh

```diff
@@ -204,3 +204,10 @@ function set_upgrade_property() {
 	source_upgrade_properties ||
 		die "failed to read properties file after setting '$1=$2'"
 }
+
+function turn_verbose_logging_on() {
```

Comment:
Might be useful to print a message here to inform that debug info is available in `/var/log/syslog`.